sale_procurement_group_by_line/model/sale.py

```diff
@@ -43,7 +43,7 @@ def _action_launch_stock_rule(self, previous_product_uom_qty=False):
             if (
                 line.state != "sale"
                 or line.order_id.locked
-                or line.product_id.type not in ("consu", "product")
+                or line.product_id.type != "consu"
```

Comment:
I'm wondering if we should test `is_storable` field

Reply:
Hi @rousseldenis, I see Odoo does this

[sale_stock/models/sale_order_line.py#L357](https://github.com/odoo/odoo/blob/be0d490f6f541a965ce8ca6d5a395f56fc6ad8cf/addons/sale_stock/models/sale_order_line.py#L357)

Although is_storable=False implies no inventory tracking, consumable products (type: consu, is_storable: False) may still require provisioning rules.

Do you agree?